ui/selectors/confirm-transaction.js

```diff
@@ -230,13 +230,16 @@ export const transactionFeeSelector = function (state, txData) {
     gasPrice = getAveragePriceEstimateInHexWEI(state) || '0x0';
   }
 
-  const fiatTransactionAmount = getValueFromWeiHex({
-    value,
-    fromCurrency: nativeCurrency,
-    toCurrency: currentCurrency,
-    conversionRate,
-    numberOfDecimals: 2,
-  });
+  const fiatTransactionAmount = conversionRate
```

Comment:
![image](https://user-images.githubusercontent.com/4448075/122821748-d7ae6e80-d2a2-11eb-8a23-3eed5db0effa.png)
left: this PR, "Fiat" selected for primary, getConversionRate set to return undefined.

My assumption would be that, when the conversionRate is undefined, that the original UI (right) is shown but without the $0.00?